core/geometry/src/test/curve/RegionBoolean.test.ts

```diff
@@ -242,35 +250,274 @@ describe("RegionBoolean", () => {
+        for (const g of section){
+          if (g instanceof LineString3d) {
+            if (g.packedPoints.front()?.isAlmostEqualMetric(g.packedPoints.back()!)) {
+              closedAreas.push(g);
+              }
+            }
+        }
+        GeometryCoreTestIO.captureCloneGeometry(allGeometry, section, x0, y0);
+        y0 += dy;
+        GeometryCoreTestIO.captureCloneGeometry(allGeometry, closedAreas, x0, y0);
+        y0 += dy;
+        for (let i = 0; i < closedAreas.length; i++){
+          for (let j = i + 1; j < closedAreas.length; j++){
+            const pairs = CurveCurve.closeApproachProjectedXYPairs(closedAreas[i], closedAreas[j], minSearchDistance);
+            y0 += dy;
+            GeometryCoreTestIO.captureCloneGeometry(allGeometry, closedAreas[i], x0, y0);
+            GeometryCoreTestIO.captureCloneGeometry(allGeometry, closedAreas[j], x0, y0);
+            let numPoint = 0;
+            let shortestPair: CurveLocationDetailPair | undefined;
+            let shortestDistance = Number.MAX_VALUE;
+            let numOverlap = 0;
+            for (const p of pairs) {
+              if (p.detailA.fraction1 !== undefined)
+                numOverlap++;
+              if (p.detailA.point.isAlmostEqual(p.detailB.point)){
+                GeometryCoreTestIO.createAndCaptureXYCircle(allGeometry, p.detailA.point, 0.05, x0, y0);
+                numPoint++;
+              } else {
+                const d = p.detailA.point.distance(p.detailB.point);
+                if (d < shortestDistance) {
+                  shortestDistance = d;
+                  shortestPair = p;
+                }
+              }
+            }
+            if (numOverlap > 0)
+              console.log({ numOverlap, i, j});
+            if (shortestPair && numPoint === 0)
+            for (const p of pairs) {
+              if (p.detailA.point.isAlmostEqual(p.detailB.point)){
+              } else {
+                const d = p.detailA.point.distance(p.detailB.point);
+                if (d < distanceDisplayFactor * shortestDistance) {
+                  const arc = constructEllipseAroundSegment(p.detailA.point, p.detailB.point, 1.2, 0.2);
+                  GeometryCoreTestIO.captureGeometry(allGeometry, arc, x0, y0);
+                  GeometryCoreTestIO.captureCloneGeometry(allGeometry, [p.detailA.point, p.detailB.point], x0, y0);
+                  }
+              }
+            }
+          }
+        }
+        x0 += 100.0;
+      }
+    GeometryCoreTestIO.saveGeometry(allGeometry, "RegionBoolean", "SectioningLineStringApproach");
+      expect(ck.getNumErrors()).equals(0);
+      });
+      //  in the input directories ...
+      //    Madhav.json is the whole input set for 5 sections.
+      //    sectionA..sectionE are splits of the separate sections.
+      //  Note that the corresponding .imjs files are reduced to linestrings and are accessed in other tests.
+      it.skip("SectioningJson", () => {

```

Comment:
Intentionally skipped?

Reply:
yes.  At the moment its purpose is part of helping the AI road section recognition developers figure out what's in their data.  It's console output is useful in nudging that forward but is not testing anything in the library.   As we understand more of what their fixup needs are that might evolve into tests of the ChainCollector.